Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -105,6 +105,18 @@ + (void)application:(__unused UIApplication *)application didReceiveRemoteNotifi
                                                     userInfo:notification];
 }
 
++ (void)application:(__unused UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error
+{
+  NSDictionary *errorInfo = @{
+                              @"error" : @TRUE,
+                              @"description" : [error localizedDescription],
+                              @"reason" : [error localizedFailureReason]
+                              };
+  [[NSNotificationCenter defaultCenter] postNotificationName:RCTRemoteNotificationsRegistered
```

Comment:
We have a standard object format for errors, which you can generate using the RCTJSErrorFromNSError() function from RCTUtils.h